gcp_variant_transforms/pipeline_common.py

```diff
@@ -78,6 +78,9 @@ def parse_args(argv, command_line_options):
   if hasattr(known_args, 'input_pattern') or hasattr(known_args, 'input_file'):
     known_args.all_patterns = _get_all_patterns(
         known_args.input_pattern, known_args.input_file)
+  # Fix NameErrors:
+  # https://cloud.google.com/dataflow/docs/resources/faq#how_do_i_handle_nameerrors
+  pipeline_args.extend(['--save_main_session', 'True'])
```

Comment:
Do we need to mention this in our documentations? Or users don't need to know about this at all?